packages/lexical-devtools/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx"
+  },
```

This can probably just inherit from the base `tsconfig` to avoid having different TS configs per package.

```suggestion
  "compilerOptions": {
    "strict": true,
  },
  "extends": "../tsconfig.json" 
```

---

> This can probably just inherit from the base tsconfig to avoid having different TS configs per package.

Sounds good, will commit